source/_data/SymbioticLab.bib

```diff
@@ -767,6 +767,7 @@ @InProceedings{tpp:asplos23
   year      = {2023},
 
   publist_confkey = {ASPLOS'23},
+  publist_badge = {IEEE MICRO Top Picks Honorable Mention},
```

Comment:
```suggestion
  publist_badge = {MICRO Top Picks Honorable Mention},
```
I believe this will do it. You can click "commit" on the comment to apply this change.